services/director-v2/src/simcore_service_director_v2/modules/db/repositories/comp_runs.py

```diff
@@ -145,6 +145,22 @@ async def get(
                 raise ComputationalRunNotFoundError
             return CompRunsAtDB.model_validate(row)
 
+    async def get_latest_run_by_project(
+        self,
+        project_id: ProjectID,
+    ) -> CompRunsAtDB:
+        async with pass_or_acquire_connection(self.db_engine) as conn:
+            result = await conn.execute(
+                sa.select(comp_runs)
+                .where(comp_runs.c.project_uuid == f"{project_id}")
+                .order_by(desc(comp_runs.c.run_id))
```

Comment:
MINOR: for such small functions, i would keep the namespace `sql.desc`  (provided `from sqlalachem import sql`